### DISCUSSION TITLE
How to resize images?

### DISCUSSION THREAD

**Discussion by biphasic** [CaiJimmy/hugo-theme-stack#780] [Q&A] [answered]
Hello,
unfortunately https://stack.jimmycai.com/writing/markdown#writing only specifies how to add images, without saying how it's possible to resize them. 
Normally things like `<img src="portrait.jpeg" alt="Name" height="180px"/>` work in Markdown and I can also see it correctly rendered in the Markdown. But the theme doesn't seem to be able to pick it up, it just doesn't display anything. 

Thanks so much!

**Comment by CaiJimmy** [comment, marked as answer]
There's currently no support for resizing images, sorry.

**Comment by biphasic** [comment]
Thanks for the quick response! 

Le 13 février 2023 19:55:29 GMT+00:00, Jimmy Cai ***@***.***> a écrit :
>There's currently no support for resizing images, sorry.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/CaiJimmy/hugo-theme-stack/discussions/780#discussioncomment-4962498
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Envoyé de mon appareil Android avec Courriel K-9 Mail. Veuillez excuser ma brièveté.
